test/functional/services/snapshots.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from 'expect.js';
+import { dirname, resolve } from 'path';
+import { writeFile, readFileSync } from 'fs';
+import { fromNode as fcb, promisify } from 'bluebird';
+import mkdirp from 'mkdirp';
+import del from 'del';
+
+const writeFileAsync = promisify(writeFile);
+
+export async function SnapshotsProvider({ getService }) {
+  const log = getService('log');
+  const config = getService('config');
+
+  const SESSION_DIRECTORY = resolve(config.get('snapshots.directory'), 'session');
+  const BASELINE_DIRECTORY = resolve(config.get('snapshots.directory'), 'baseline');
+  await del([SESSION_DIRECTORY]);
+
+  class Snapshots {
+
+    /**
+     *
+     * @param name {string} name of the file to use for comparison
+     * @param updateBaselines {boolean} optional, pass true to update the baseline snapshot.
+     * @return {Promise.<number>} Percentage difference between the baseline and the current snapshot.
```

Comment:
Looks like this just returns 0 no matter what

Reply:
ah yeah, too much copy paste from screenshots service :dagger: